eng/common/pipelines/templates/steps/policheck.yml

```diff
@@ -0,0 +1,36 @@
+
+parameters:
+  ExclusionDataBaseFileName: ''
+  TargetDirectory: ''
+  PublishAnalysisLogs: false
+  PoliCheckBlobSAS: '$(azuresdk-policheck-blob-SAS)'
```

Comment:
Do we need to add variable group 95 to the pipeline generators for this?

Reply:
How did you determine the number 95?

Reply:
I was wondering that as well :) I assume he looked up the variable group that defines this.

Reply:
I tried that, I still didn't see any number anywhere.

Reply:
The only way I have found to find the variable group id is to look at the URL when you are viewing a variable group.

Reply:
Thanks @weshaggard . You can also find it using azure cli `az pipelines variable-group list --group-name "azuresdkartifacts SAS tokens`. But Url is definitly easier.